cmd/sgx_plugin/README.md

```diff
@@ -85,83 +79,58 @@ the devel images from Docker Hub by default.
 
 `<RELEASE_VERSION>` needs to be substituted with the desired release version, e.g. `v0.19.0` or main.
 
-#### Deploy node-feature-discovery
+### Installation Using the Operator
+
+First, deploy `node-feature-discovery`:
 
 ```bash
 $ kubectl apply -k https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/nfd/overlays/sgx?ref=<RELEASE_VERSION>
 $ kubectl apply -k https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/nfd/overlays/node-feature-rules?ref=<RELEASE_VERSION>
 ```
 
 **Note:** The [default configuration](/deployments/nfd/overlays/node-feature-rules/node-feature-rules.yaml) assumes that the in-tree driver is used and enabled (`CONFIG_X86_SGX=y`). If
-the SGX DCAP out-of-tree driver is used, the `kernel.config` match expression in must be removed.
+the SGX DCAP out-of-tree driver is used, the `kernel.config` match expression must be removed.
 
-#### Deploy Intel Device plugin operator
+Next, deploy the Intel Device plugin operator:
 
 ```bash
 $ kubectl apply -k https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/operator/default?ref=<RELEASE_VERSION>
 ```
 
 **Note:** See the operator [deployment details](/cmd/operator/README.md) for its dependencies and for setting it up on systems behind proxies.
 
-#### Deploy SGX device plugin with the operator
+Finally, deploy the SGX device plugin with the operator
```

Comment:
I guess also other plugins should eventually get their own operator installation section, if operator installation is plugin specific (=> file bugs against plugins?), or this should move later to some common document, if it's not?

Reply:
I tried to focus on moving everything "developer" related to elsewhere and thought we should get back to deployment options later.

Reply:
moved as a tracking item in #1045